config/vufind/sitemap.ini

```diff
@@ -43,7 +57,7 @@ fileLocation   = /tmp
 ;    record ID in order to generate a link. If this setting is omitted, the
 ;    default will be "Solr,/Record/"
 index[] = "Solr,/Record/"
-;index[] = "SolrAuth,/Authority/Record?id="
+;index[] = "SolrAuth,/AuthorityRecord/"
```

Comment:
Good catch! I'll try to remember to backport this fix since it would be useful to fix this outdated example sooner rather than later.

Reply:
OK, so i added a reference to the Wiki page and moved most of the notes to the section _Best Practices_ => _Understanding Crawling Budgets_ in the Wiki (maybe the exact VuFind Version should be mentioned there when lastmod will be available after this PR has been merged.)

Reply:
I pushed this change into the release-9.1 branch since it's essentially a bug fix (and only affects a comment, so it's harmless). I've merged dev back into this branch as well to get everything back into sync.